### DISCUSSION TITLE
FHIR Extension Installation Issue

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#4553] [Q&A] [answered]
Hi,

I just downloaded the latest FHIR extension and I am trying to install it on a 3.1.0 instance and I am receiving the following error:

.....
Caused by: com.mirth.connect.client.core.EntityException: {
url: "/api/3.10.1/extensions/_install",
status: "500",
message: "Request failed.",
servlet: "org.glassfish.jersey.servlet.ServletContainer-1ab8e4d5",
}.....

I have never installed an extension before and I am using the Extension -> Install option in Mirth connect itself.   The instance is running under a domain account, which should have full rights to the whole server, so I am not sure if this is a rights issue or something else is going on.   I also downloaded a previous FHIR extension version and I have the same issue.   Any help would be be appreciated.

**Comment by tonygermano** [comment, marked as answer]
Probably this: https://github.com/nextgenhealthcare/connect/issues/4437

If you are using 3.10.0, upgrade to 3.10.1 or 3.11.0.

**Reply by rjkroll** [reply]
Thank you.   The fix was indeed to upgrade to 3.11.0
